integration_tests/test_import_05.py

```diff
@@ -0,0 +1,3 @@
+from test_import.sys import hi_from_user_sys
+
+hi_from_user_sys()
```

Comment:
And then assert here. So that we know that we are not having any hidden bugs.

Reply:
Done! Thanks for the guidance!